src/components/CreatePost.js

```diff
@@ -35,12 +37,12 @@ export default function CreatePost({ refresh, setRefresh }) {
   function submitData(e) {
     e.preventDefault();
     setDisabled(true);
-    if (form.url === "") {
-      alert("Insira o link que deseja compartilhar");
+    if (form.url === "Please enter a valid URL") {
+      alert("");
```

Comment:
Why is the alert with an empty string?

Reply:
I believe she meant:

if ( form.url === "" ) {
      alert("Please enter a valid URL");
}

Reply:
Oooop, sorry. You got it, Guilherme. Already fixed it.